src/ResourceManager/HDInsight/Commands.HDInsight.Test/ScenarioTests/DataLakeStoreScenarioTests.cs

```diff
@@ -10,11 +10,18 @@ public DataLakeStoreScenarioTests(Xunit.Abstractions.ITestOutputHelper output)
             ServiceManagemenet.Common.Models.XunitTracingInterceptor.AddToContext(new ServiceManagemenet.Common.Models.XunitTracingInterceptor(output));
         }
 
-        [Fact]
+        [Fact (Skip = "Test requires fixing GraphAPI by upgrading to latest test framework")]
```

Comment:
The solution is to update the version of graph rbac manageemnt client you are using.  You cna find examples of constructiung the new client here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1313/

You should also be calling SetupManagementClients instead of SetupSomeOfManagementClients, this will make the test fail fast before runtime if client conbstruction fails.
Please ping us, or att